.github/workflows/tests.yaml

```diff
@@ -5,6 +5,8 @@ on:
       branches: [ main ]
   pull_request:
   workflow_dispatch:
+  release:
+    types: [published]
```

Comment:
hypothetically makes it so that whenever wheels.yaml is run, test.yaml will have to complete before anything else happens